### PR TITLE
Add an option to skip the TOC check on refine

### LINKED ISSUE #159: refine - add a TOC mismatch override option
As some generic drives return mismatching TOC data.

### PR DESCRIPTION
**Pull request by Sazpaimon**
Fixes #159

### CODE REVIEW

File: options.ixx [hunk 1]
```diff
@@ -329,6 +333,7 @@ export struct Options
         LOG("\t--dump-read-size=VALUE         \tnumber of sectors to read at once on initial dump, DVD only (default: {})", dump_read_size);
         LOG("\t--overread-leadout             \tdo not limit lead-out to the first hundred sectors, read until drive returns SCSI error");
         LOG("\t--force-unscrambled            \tdo not attempt to read data sectors as audio (BE read method only)");
+        LOG("\t--skip-toc-check               \tdo not check TOC when refining a disc");
```
Comment (superg) [inline options.ixx:336]: Please rename an option to --force-refine and variable name to force_refine respectively.

**Review by superg — CHANGES_REQUESTED** [review]
Sorry I totally forgot about opened PR's.

File: options.ixx (hunk 1 above)
Comment (Sazpaimon) [reply options.ixx]: changed

**Review by superg — APPROVED** [review]
All good, thanks!

**Pull request merged**